pandas-stubs/core/arrays/datetimelike.pyi

```diff
@@ -67,16 +67,7 @@ class DatetimeLikeArrayMixin(ExtensionOpsMixin, ExtensionArray):
     def resolution(self): ...
     __pow__ = ...
     __rpow__ = ...
-    __mul__ = ...
     __rmul__ = ...
-    __truediv__ = ...
-    __rtruediv__ = ...
-    __floordiv__ = ...
-    __rfloordiv__ = ...
-    __mod__ = ...
-    __rmod__ = ...
-    __divmod__ = ...
-    __rdivmod__ = ...
```

Comment:
do parent classes already declare these methods or why are they removed?

Reply:
The subclasses of `DatetimeLikeArrayMixin` implement those methods, so `mypy` or `pyright` (can't remember which) was reporting inconsistencies between having the `...` and the definitions of the methods.  I deleted the ones that had conflicts.